web-module/src/main/webapp/resources/structEditJS/initStructEditor.js

```diff
@@ -26,17 +26,38 @@
    */
 
   var initStructEditorContentChangedHandler = function(event) {
-    var _me = this;
     var checkRoot = event.memo.htmlElem || document.body;
     console.log('initStructEditorContentChangedHandler: run for ', checkRoot);
     $(document.body).fire('cel:initStructEditor', { 'checkRoot' : checkRoot });
     console.log('initStructEditorContentChangedHandler: finish for ', checkRoot);
   };
+  
+  var initTinyMCE = function(event) {
+    tinymce.init({
+      selector: 'textarea.tinyMCE',
+      height: 500,
+      theme: 'modern',
+      plugins: 'print preview searchreplace autolink directionality visualblocks visualchars fullscreen image link media template codesample table charmap hr pagebreak nonbreaking anchor toc insertdatetime advlist lists textcolor wordcount imagetools contextmenu colorpicker textpattern help',
+      toolbar1: 'formatselect | bold italic strikethrough forecolor backcolor | link | alignleft aligncenter alignright alignjustify  | numlist bullist outdent indent  | removeformat',
+      image_advtab: true,
+      templates: [
+        { title: 'Test template 1', content: 'Test 1' },
+        { title: 'Test template 2', content: 'Test 2' }
+      ],
+      content_css: [
+        '//fonts.googleapis.com/css?family=Lato:300,300i,400,400i',
+        '//www.tinymce.com/css/codepen.min.css'
+      ]
+     });
+  };
 
   celAddOnBeforeLoadListener(function() {
     window.celStructEditorManager = new CELEMENTS.structEdit.StructEditorManager();
     $(document.body).stopObserving("celements:contentChanged", initStructEditorContentChangedHandler);
     $(document.body).observe("celements:contentChanged", initStructEditorContentChangedHandler);
+    $(document.body).stopObserving("celements:contentChanged", initTinyMCE);
+    $(document.body).observe("celements:contentChanged", initTinyMCE);
```

Comment:
only observer on the element related to the tinyMCE editor. It must be possible that one editor is loaded async later than a first one and the tinyMCE in the first editor must not be reinitialized because of the contentChanged issued in the second part.